scheduler/scheduler.py

```diff
@@ -0,0 +1,46 @@
+"""
+scheduler.py
+Module to send the daily reminders using the apscheduler.
+"""
+import httpx
+from fastapi import HTTPException
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+from oauth import create_access_token
+from settings import app_url
+from scheduler.helpers import get_tz_list
+
+ADMIN_TOKEN = create_access_token(data={"user_id": 1, "user_role": "admin"})
+URL = f"{app_url}:8000/api/v1/users/send_reminders"
+HEADERS = {
+    "Authorization": f"Bearer {ADMIN_TOKEN}"
+}
+scheduler = AsyncIOScheduler()
+
+
+async def send_reminders_to_users(timezone, client):
```

Comment:
Do you need `client` parameter in the `send_reminders_to_user` function?

Also you are redefining `client` in the 
`async with httpx.AsyncClient() as client:`
its better use different names because redefining could lead to unexpected results